src/Controller/API/Administrator/EducationLevelAPI.php

```diff
@@ -1,47 +1,89 @@
 <?php
 
+/**
+ * This file is part of ESO system.
+ *
+ * (c) SBSystem 2018
+ *
+ * For the full license information, please see LICENSE.md at https://github.com/SBSystem/ESO/LICENSE.md
+ */
+
 namespace App\Controller\API\Administrator;
 
+use App\AppBundle\Factory\Creators\EducationLevelCreator;
 use App\Controller\API\Interfaces\CRUD;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
-class EducationLevelAPI implements CRUD
+class EducationLevelAPI extends Controller implements CRUD
 {
     /**
      * This function is used to create education level
      */
     public function create()
     {
-        $request = new Request();
+        $jsonResponse = new JsonResponse();
 
-        $className = $request->request->get('class_name');
-        $classLevel = $request->request->get('class_level');
-        $educatorId = $request->request->get('educatorId');
+        if(isset($_POST['className']) && isset($_POST['classLevel']) && isset($_POST['educatorId'])){
```

Comment:
Nie czaję, pobierałeś ładnie posta w ludzki sposób, a potem zmieniłeś to na takie guano. W Symfony nie używa się tablic superglobalnych bezpośrednio. Od tego jest klasa Request żeby z niej korzystać.